src/en/developer-resources.md

```diff
@@ -1,25 +1,24 @@
-Title: Writing charms using resources
+Title: Resources | Developer guide
 
-# Writing charms that use resources
+# Developer resources
```

Comment:
They aren't really 'developer resources'. Perhaps 'Using resources' would be better

Reply:
OK

Reply:
Should I leave the menu alone (*Juju Resources*)?

Reply:
the menu item is ok